### PR TITLE
fix(falco): reintroduce service account

### PR DESCRIPTION
**Pull request by alacuku**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

 /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

 /area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

> /area event-generator-chart

> /area k8s-metacollector

**What this PR does / why we need it**:

This PR reintroduces the service account removed in the previous release. see #612 

**Which issue(s) this PR fixes**:
#612 

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: charts/falco/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: falco
-version: 4.0.0
+version: 4.0.1
```
Comment (cpanato) [inline charts/falco/Chart.yaml:3]: should be a minor version

File: charts/falco/templates/pod-template.tpl [hunk 2]
```diff
@@ -16,6 +16,7 @@ metadata:
       {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
+  serviceAccountName: {{ include "falco.serviceAccountName" . }}
```
Comment (cpanato) [inline charts/falco/templates/pod-template.tpl:19]: if we dont set that it will get the default one?

**Review by cpanato — CHANGES_REQUESTED** [review]

File: charts/falco/templates/pod-template.tpl (hunk 2 above)
Comment (alacuku) [reply charts/falco/templates/pod-template.tpl:19]: Correct.

**Comment by alacuku** [comment]
Hey @cpanato, could you please have a look?

**Review by cpanato — APPROVED** [review]
/lgtm

thanks

**Comment by CarpathianUA** [comment]
Hi @alacuku 
Thank you, I think yes. I'll test a new chart release a little bit later, but PR changes looks good to me!

**Pull request merged**